examples/hal/compute/main.rs

```diff
@@ -46,7 +51,19 @@ fn main() {
         .open_with::<_, Compute>(1, |_family| true)
         .unwrap();
 
-    let shader = device.create_shader_module(include_bytes!("shader/collatz.spv")).unwrap();
+    let mut glsl = String::new();
+    File::open("compute/shader/collatz.comp")
+        .unwrap()
+        .read_to_string(&mut glsl)
+        .unwrap();
+    let mut spirv: Vec<u8> = vec![];
+    for byte in glsl_to_spirv::compile(&glsl, glsl_to_spirv::ShaderType::Compute)
+        .unwrap()
+        .bytes()
+    {
+        spirv.push(byte.unwrap())
```

Comment:
this shouldn't be as complex. Maybe just `unwrap().bytes().collect::<Vec<_>>()`?